HW03/veaceslav_irizanov/veaceslav_irizanov_hw_03_t_02.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class Homework3
+  def task2(log)
```

Comment:
- move the regular expression to constants
- You have a complex solution, so you should refactor the code to make it readable:

1) Create 3 regular expressions to select DATETIME, IP, ADDRESS
2) Split the log you made on line 5
3) Select all lines that have a POST request
4) use an iterator on selected lines with POST request and collect datetime, ip  and address using your regex
5) the last line in the iterator block will look like `"#{datetime} FROM: #{ip} TO: #{address}"`